application/config/config.php

```diff
@@ -5,18 +5,9 @@
 // overwritten, or superseded by `config/(development|production/...php`
 // This file is in force on docker-compose and cloud.gov platforms
 
-// TODO: Why are project_shared_path and root_dir
-// determined differently?
-//   $root_dir = '{{ current_source_symlink }}';
-$root_dir = dirname(__DIR__, 2);
-
-/**
- * Use Dotenv to set required environment variables and load .env file in root
- */
-if (file_exists($root_dir . '/.env')) {
-    $dotenv = new \Dotenv\Dotenv($root_dir);
-    $dotenv->load();
-}
+// This isn't actually used... We already got access to DotEnv in bootstrap.php
```

Comment:
In ideal circumstances we'd just let CodeIgniter's built-in composer support do all the work, which is triggered by this config option. However, we already sourced composer's `autoload.php` prior to this in `bootstrap.php` in order to get DotEnv/.env vars available for configuration purposes in this file. 

We don't _really_ want all those composer modules accessible by this point, but I guess working through my composer-newb situation I'm only just realizing that [composer already does lazy-loading](https://www.reddit.com/r/PHP/comments/29zh7j/why_doesnt_composer_autoloading_significantly/). I left this here as a signpost for the future, but it's probably better to just take it out to avoid confusion; will do so once I know that's the right move, which I should be able to confirm during the move to the [composer-installed CodeIgniter](https://github.com/kenjis/codeigniter-composer-installer), which is coming right up.